spec/requests/api/v1/items_request_spec.rb

```diff
@@ -64,4 +64,23 @@
       expect(items).to_not have_key(:item)
     end
   end
+
+  it "sends one item" do
+    merchant = FactoryBot.create(:merchant)
+    item = FactoryBot.create(:item, merchant_id: merchant.id)
```

Comment:
If I'm understanding your comment on the overall PR correctly.... 

I tested this out in pry and I believe this would still create the merchant and item:
```ruby
FactoryBot.create(:merchant)
FactoryBot.create(:item, merchant_id: merchant.id)
```
However the reason we want to assign them to "merchant" and "item" variables is so we can reference them later. For example, if we were to not set the merchant variable in line 69, then we wouldn't be able to reference `merchant.id` when we set that in line 70.

I may have misunderstood where your confusion was coming from, but I hope that provides some clarity maybe?